providers/azure.go

```diff
@@ -361,6 +369,49 @@ func (p *AzureProvider) getEmailFromProfileAPI(ctx context.Context, accessToken
 	return getEmailFromJSON(json)
 }
 
+func getUserFromJSON(json *simplejson.Json) (string, error) {
+	var user string
+	var err error
```

Comment:
Nit: Neither of these are needed as they are initialised inline (or can be)

Reply:
I'll remove err for sure but `user` is assigned 3 times in that function, is it better to have it as in-line in first and then go re-assign it or should I just leave that as is. appreciate your thoughts